contracts/src/lido/LidoGatewayManager.sol

```diff
@@ -0,0 +1,238 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.8.16;
+
+import {ScrollGatewayBase} from "../libraries/gateway/ScrollGatewayBase.sol";
+
+// solhint-disable func-name-mixedcase
+
+abstract contract LidoGatewayManager is ScrollGatewayBase {
```

Comment:
Why inherit from `ScrollGatewayBase`? For `onlyOwner`?

Reply:
Yes, for `onlyOwner`.